includes/RestApi/SitePagesController.php

```diff
@@ -105,7 +105,7 @@ private function set_homepage( $homepage_pattern_slug ) {
 			\update_option( Options::get_option_name( 'show_on_front', false ), 'page' );
 		}
 
-		$post_id = $this->publish_page( 'Homepage', $pattern_data['content'], true, $pattern_data['meta'] );
+		$post_id = $this->publish_page( 'Home', $pattern_data['content'], true, $pattern_data['meta'] );
```

Comment:
```suggestion
		$post_id = $this->publish_page( __( 'Home', 'wp-module-onboarding' ), $pattern_data['content'], true, $pattern_data['meta'] );
```
I think we can pass this to the `__` i18n function.